core/designsystem/src/main/kotlin/org/mifos/mobile/core/designsystem/theme/Theme.kt

```diff
@@ -50,6 +55,15 @@ fun MifosMobileTheme(
         else -> LightThemeColors
     }
 
+    val view = LocalView.current
+    if (!view.isInEditMode) {
+        SideEffect {
+            val window = (view.context as Activity).window
+            window.statusBarColor = colors.primary.toArgb()
+            WindowCompat.getInsetsController(window, view).isAppearanceLightStatusBars = !useDarkTheme
+        }
+    }
```

Comment:
Use `enableEdgeToEdge()` API, and set status bar color to **Transperant** in both light and dark theme.